datalad/support/gitrepo.py

```diff
@@ -397,21 +440,23 @@ class GitProgress(WitlessProtocol):
         CHECKING_OUT: ("Check out", "Things"),
     }
 
-    __slots__ = ('_seen_ops', '_pbars', '_encoding')
+    __slots__ = ('_unprocessed', '_seen_ops', '_pbars')
```

Comment:
I am surprised that it didn't error out before somehow since I was expecting that if `__slots__` specified then assignment of unknown ones to lead to an error. oh well

Reply:
`GitProgress` inherits from `WitlessProtocol`, which does not use `__slots__`, and so `GitProgress` will still have a `__dict__` attribute and support assignment of arbitrary attributes.